script/script.js

```diff
@@ -0,0 +1,177 @@
+	var confirm = document.querySelector("#signupform input:nth-child(7)").value;
+	if(confirm.length < 8) {
+		isItTrue.someval4(submitButtonEnabled);
+		valildateAll("Password should be more than 8 characters long", nthChild);
+	}
+	else {
+		submitButtonEnabled = true;
+		isItTrue.someval4(submitButtonEnabled);
+		document.querySelector("#signupform span:nth-child(" + nthChild + ")").innerHTML = '';
+	}
+	
+	if(confirm !== password) {
+		isItTrue.someval4(false);
+		valildateAll("Passwords not matching", nthChild);
+	}
+	else {
+		submitButtonEnabled = true;
+		document.querySelector("#signupform span:nth-child(" + nthChild + ")").innerHTML = '';
+		isItTrue.someval4(submitButtonEnabled);
+	}
+	enableSubmit();
+}
+
+function valildateAll(displayWarning, nthChild) {
+	//document.getElementsByClassName("modal_heading")[0].innerHTML = displayWarning;
+	document.querySelector("#signupform span:nth-child(" + nthChild + ")").style.color = "red";
+	document.querySelector("#signupform span:nth-child(" + nthChild + ")").style.fontSize = "13px";
+	document.querySelector("#signupform span:nth-child(" + nthChild + ")").innerHTML = displayWarning;
+}
+
+let isItTrue = function() {
+	value1 = false;
+	value2 = false;
+	value3 = false;
+	value4 = false;
+	return {
+		someval1: function(val) {
+			value1 = val;
+		},
+		getSomeVal1: function() {
+			return value1;
+		},
+		someval2: function(val) {
+			value2 = val;
+		},
+		getSomeVal2: function() {
+			return value2;
+		},
+		someval3: function(val) {
+			value3 = val;
+		},
+		getSomeVal3: function() {
+			return value3;
+		},
+		someval4: function(val) {
+			value4 = val;
+		},
+		getSomeVal4: function() {
+			return value4;
+		}
+	}
+}();
+
+function enableSubmit() {
+	if(isItTrue.getSomeVal1() && isItTrue.getSomeVal2() && isItTrue.getSomeVal3() && isItTrue.getSomeVal4()) {
+		document.querySelector("#signupform button").disabled = "false";
+		document.querySelector("#signupform button").style.cursor = "pointer";
+		document.querySelector("#signupform button").style.background = "#1DABDD";
+	}
+	else {
+		document.querySelector("#signupform button").disabled = "true";
+		document.querySelector("#signupform button").style.cursor = "default";
+		document.querySelector("#signupform button").style.background = "lightgrey";
+	}
+}
+
+let loginUser = function(event) {
+	event.preventDefault();
+	let username = document.forms["signinform"]["username"].value;
+	let password = document.forms["signinform"]["password"].value;
+
+	if(username === '' || password === '') {
+		document.querySelector("#loginMessage").style.color = "red";
+		document.querySelector("#loginMessage").innerHTML = "Fields cannot be left empty";
+	}
+	else {
+		document.querySelector("#loginMessage").innerHTML = '';
+		let user = {user_email: username, user_pass: password};
+		var init = {
+			method: "POST",
+			body: JSON.stringify(user)
+		}
+		var resp = fetch('https://fsd1.herokuapp.com/users/login',
```

Comment:
Add a .catch error handler in case the API call itself fails. This applies to all fetch requests(promises) that you create.